setup/ansible/roles/userland/files/aarch64.patch

```diff
@@ -71,4 +71,3 @@ index 1c966b8..757986c 100644
 
  //#define EXTRACT_BYTE(x, n)	((unsigned long long)((uint8_t *)&x)[n])
  // xxx work around a compiler bug...
```

Comment:
There needs to be a newline at the end of this file or `git apply patch` will fail.

Reply:
I'm confused here. thought I added a newline but it looks to be removed. Reverting change.

Reply:
Fixed